### DISCUSSION TITLE
Repository is not found

### DISCUSSION THREAD

**Discussion by esetnik** [renovatebot/renovate#12203] [Request Help]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Describe the bug

Renovate was working previously but suddenly I get this error with no config change. I tried removing renovate app from Github and adding it back again, no jobs have run in last two days.

![image](https://user-images.githubusercontent.com/664434/137750248-46f7d0e9-4bf7-4743-9372-73301b139545.png)

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: No dangling containers to remove
INFO: Repository started
{
  "renovateVersion": "28.1.4"
}
DEBUG: Using localDir: /mnt/renovate/gh/ProdigyEMS/prodigy
DEBUG: Repository cache is valid
DEBUG: initRepo("ProdigyEMS/prodigy")
DEBUG: Overriding default GitHub endpoint
{
  "endpoint": "https://api.github.com/"
}
DEBUG: Caught initRepo error
ERROR: Repository is not found
DEBUG: Repository timing splits (milliseconds)
{
  "splits": {},
  "total": 238
}
DEBUG: http statistics
{
  "urls": {
    "https://api.github.com/graphql (POST)": 1
  },
  "hostStats": {
    "api.github.com": {
      "requestCount": 1,
      "requestAvgMs": 139,
      "queueAvgMs": 0
    }
  },
  "totalRequests": 1
}
INFO: Repository finished
{
  "durationMs": 238
}```

</details>


### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
Something went funny with GitHub's API - maybe an accidental app blocking, for example - and a subset of repos were marked as removed/404 not found. We've queued up every repo for a retry just now (including forks, archived repos, etc). I think anyone with an existing PR or dependency dashboard can hopefully request a job on demand in the meantime using a checkbox.

**Comment by dschaller** [comment]
I noticed we are also having the same issue in github.com/lyft/clutch but we haven't had a PR opened in ~27 days

[renovate config](https://github.com/lyft/clutch/blob/main/.github/renovate.json)

Renovate Logs
```bash
DEBUG: No dangling containers to remove
INFO: Repository started
{
  "renovateVersion": "28.1.4"
}
DEBUG: Using localDir: /mnt/renovate/gh/lyft/clutch
INFO: Repository cache invalidated
DEBUG: initRepo("lyft/clutch")
DEBUG: Overriding default GitHub endpoint
{
  "endpoint": "https://api.github.com/"
}
DEBUG: Caught initRepo error
ERROR: Repository is not found
```

**Comment by edalex-ian** [comment]
Yup, we also started having the issue (across our four registered repos) about 19 hours ago. And even though I have a PR with the checkbox ticked (as suggested above) nothing seems to be happening:

![image](https://user-images.githubusercontent.com/43919233/138002570-20630d27-edc1-47a2-a2c8-70d2dcd0c5d5.png)

![image](https://user-images.githubusercontent.com/43919233/138002634-b0691460-1a43-460a-8c36-d4478a0b16ea.png)

**Comment by edalex-ian** [comment]
Aww man, sorry, just as I submitted that I got notifications that it's suddenly doing stuff again - although with a new unexpected status (repository changed), but hey:

![image](https://user-images.githubusercontent.com/43919233/138002968-97f0c5d5-c624-43ed-857f-c0f08c572f44.png)

**Reply by rarkins** [reply]
There seems to be a GitHub issue although I'm keeping an open mind in case it's somehow a bug in our code

**Reply by edalex-ian** [reply]
Not a problem. Looking kind of happy again on our end. :grin:

**Reply by rarkins** [reply]
FYI we reset and re-queued all repos marked as "not found" in our DB, to try to pick up on any others which were mistakenly declared as not existing by GitHub's API recently.
